content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -0,0 +1,559 @@
+      k8s.pod.name=$(OTEL_K8S_POD_NAME),
+      k8s.container.name={{ .Chart.Name }},
+      host.name=$(OTEL_K8S_NODE_NAME),
+      deployment.environment=$(OTEL_K8S_NAMESPACE)
+   {{< /code-block >}}
+
+## Run the application 
+
+To start generating and forwarding observability data to Datadog, you need to deploy the Calendar application with the OpenTelemetry SDK using Helm.
+
+1. Run the following `helm` command from the `calendar/` folder:
+```shell
+helm upgrade -i <CALENDAR_RELEASE_NAME> ./deploys/calendar/
+```
+1. This Helm chart deploys the sample Calendar application as a ReplicaSet.
+1. To test that the Calendar application is running correctly, execute the following command from another terminal window:
+   ```shell
+   curl localhost:9090/calendar 
+   ```
+1. Verify that you receive a response like:
+   ```text
+   {"date":"2022-12-30"}
+   ```
+Each call to the Calendar application results in metrics, traces, and logs being forwarded to the Datadog backend.
+
+## Explore observability data in Datadog
+
+Use the Datadog UI to explore the observability data for the sample Calendar app.
+
+### Fleet automation
+
+Explore your Datadog Agent and Collector configuration.
+
+{{< img src="/opentelemetry/embedded_collector/fleet_automation.png" alt="Review your Agent and Collector configuration from the Fleet Automation page." style="width:100%;" >}}
+
+### Live container monitoring
+
+Monitor your container health using Live Container Monitoring capabilities.
+
+{{< img src="/opentelemetry/embedded_collector/containers.png" alt="Monitor your container health from the Containers page." style="width:100%;" >}}
+
+### Infrastructure node health
+
+View runtime and infrastructure metrics to visualize, monitor, and measure the performance of your nodes.
+
+{{< img src="/opentelemetry/embedded_collector/infrastructure.png" alt="View runtime and infrastructure metrics from the Host List." style="width:100%;" >}}
+
+### Logs
+
+View logs to monitor and troubleshoot application and system operations.
+
+{{< img src="/opentelemetry/embedded_collector/logs.png" alt="View logs from the Logs Explorer." style="width:100%;" >}}
+
+### Traces
+
+View traces and spans to observe the status and performance of requests processed by your application, with infrastructure metrics correlated in the same trace. 
+
+{{< img src="/opentelemetry/embedded_collector/traces.png" alt="View traces from the Trace Explorer." style="width:100%;" >}}
+
+### Runtime metrics
+
+Monitor your runtime (JVM) metrics for your applications.
+
+{{< img src="/opentelemetry/embedded_collector/metrics.png" alt="View JVM metrics from the JVM Metrics dashboard" style="width:100%;" >}}
+
+### Collector health metrics
+
+View metrics from the embedded Collector to monitor the Collector health.
+
+{{< img src="/opentelemetry/embedded_collector/dashboard.png" alt="View Collector health metrics from the OTel dashboard." style="width:100%;" >}}
+
+## Included components
```

Comment:
Recently added 3 more components that are provided by Datadog. We should add that section at the bottom, see the word doc for reference: https://docs.google.com/document/d/1CSdVx4yVGspc8uJJqE9ZhRm7UIjHSQvOpeZFEQRBkOo/edit#bookmark=id.2mfiqne2901c

Reply:
@LutaoX - Which three components were added? The list in the document you linked appears to match what I have here. Thanks!

Reply:
Current list is up-to-date.